zerver/lib/actions.py

```diff
@@ -318,9 +351,11 @@ def delete_all_deactivated_user_sessions():
             delete_session(session)
 
 def active_humans_in_realm(realm):
+    # type: (Realm) -> List[UserProfile]
```

Comment:
Yes it returns `QuerySet`, but if we always use it as a `List`, it won't make a difference until stubs for django come out.
`QuerySet` objects are not generic, so `List` can be better for type checking.